### DISCUSSION TITLE
Is this a bug: video clipping

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#683] [General]
Whenever I clip a video in my time line it does not appear properly. I would expect the second clip to just continue the first but the time is always off and when going from the first clip to the second alway makes the timing wrong making the second clip to be in a different time. I would alway have to adjust it to fix the issue but it gets annoying. Is it possible to fix it?

**Comment by keiwando** [comment]
It's a known issue when the frame rate of the video and the timeline are different, due to the fact that the symbol's time property is a whole number:
https://github.com/keiwando/toonsquid/discussions/613

**Reply by Lioncub901** [reply]
interesting. Is it possible to change the fps of a video?
